### PR TITLE
hiiiiiiiiii

### PR DESCRIPTION
**Pull request by StarWars1980**
hiii

### CODE REVIEW

File: favorite_built_in_modules.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 #
 ###############################################################################
 import datetime
+import math
```
Comment (Hannah-sucks-at-coding) [inline favorite_built_in_modules.py:22]: nerd

**Review by Hannah-sucks-at-coding — APPROVED** [review]
I honestly don't approve of math, a number killed my father but you know what for the sake of this workshop I will approve your changes. You're welcome

File: favorite_built_in_modules.py [hunk 2]
```diff
@@ -29,7 +29,8 @@ def main():
     #
     ###########################################################################
     print("John DiMatteo likes datetime:", datetime.__doc__)
-    print("hahhaha",datetime.date)
```
Comment (KilimAnnejaro) [inline favorite_built_in_modules.py:32]: Please revert. I like to laugh 👍

Comment (StarWars1980) [reply favorite_built_in_modules.py:32]: nope

**Pull request merged**